Readme.md

```diff
@@ -101,6 +101,13 @@ var navConfigs = {
         */
         childrenProperty: 'nav_children',
 
+        /*
+        * find index.md in folder and merge it with parent folder. it is used also because plugin metalsmith-clean-urls
+        * eg: you have folder `about` and inside this folder is file `index.md`. in nav will be then only file `about/index.html` and no folder `about`
```

Comment:
Wouldn't this prevent other plugins from functioning correctly? This breaks the basic data structure of metalsmith.